src/amo/components/Search/index.js

```diff
@@ -236,8 +246,12 @@ export function mapStateToProps(state) {
   };
 }
 
+export const extractId = (ownProps) => {
+  return parsePage(ownProps.filters.page);
```

Comment:
I struggled to understand this when I first saw it. I think that this ID does not need to differentiate between component instances because the error handler gets cleared every time the search filters change, right? If so, maybe adding a comment here would be helpful to the next reader.